natsjobs/driver.go

```diff
@@ -229,6 +234,7 @@ func FromPipeline(tracer *sdktrace.TracerProvider, pipe jobs.Pipeline, log *zap.
 		deleteAfterAck:     pipe.Bool(pipeDeleteAfterAck, false),
 		deliverNew:         pipe.Bool(pipeDeliverNew, false),
 		deleteStreamOnStop: pipe.Bool(pipeDeleteStreamOnStop, false),
+		ackWait:            conf.AckWait,
```

Comment:
In this method, we don't have a configuration. You can't use `conf.AckWait` since the pipeline is created via RPC. See the roadrunner-php/jobs package.